Makefile

```diff
@@ -17,4 +17,7 @@ lint: ## Runs linting on the go files. Requires docker to be installed
 .PHONY: test
 test: ## Runs tests
 	@echo "Running tests..."
-	@go test -cover -covermode=atomic -coverprofile=coverage.out ./...
+	@go test -cover -covermode=atomic -coverprofile=coverage.tmp.out ./...
```

Comment:
Our coverage actually was heavily skewed by the 6k+ lines of generated accessor code for pointer fields. Removing this starts showing the real coverage.

Reply:
Update: We're gonna generate tests for the generated accessor methods instead.